code/modules/atmospherics/gasmixtures/auxgm.dm

```diff
@@ -71,7 +71,7 @@ GLOBAL_LIST_INIT(nonreactive_gases, typecacheof(list(GAS_O2, GAS_N2, GAS_CO2, GA
 			visibility[g] = gas.moles_visible
 			overlays[g] = new /list(FACTOR_GAS_VISIBLE_MAX)
 			for(var/i in 1 to FACTOR_GAS_VISIBLE_MAX)
-				overlays[g][i] = new /obj/effect/overlay/gas(gas.gas_overlay, i * 255 / FACTOR_GAS_VISIBLE_MAX)
+				overlays[g][i] = new /obj/effect/overlay/gas(gas.gas_overlay, i * 255 / FACTOR_GAS_VISIBLE_MAX, gas.custom_icon, gas.custom_x, gas.custom_y, gas.vis_flags, gas.layers) //austation -- added 5 new args
```

Comment:
Would appreciate a comment stating as to why, or at least in what PR :)

Reply:
for the gases to have specific layers and have specific x and y offsets, the smoke sprite is 96x96